features/step_definitions/auction_create_steps.rb

```diff
@@ -40,6 +40,10 @@
   end
 end
 
+Given(/^there is an auction with work in progress$/) do
+  @auction = FactoryGirl.create(:auction, :closed, :with_bids, :delivered)
```

Comment:
Well, delivered in this case is basically `has_delivery_url`, so I don't know if should rename to that?